### DISCUSSION TITLE
Is there an example of how to use setCalendarTo with q-date?

### DISCUSSION THREAD

**Discussion by ssuess** [quasarframework/quasar#16226] [General - Components / Directives / etc] [answered]
I have an app that uses q-date to define a `startDate` and another q-date to define `endDate`. opening them defaults to today's month of the calendar, but I want to get the `endDate` to open to the same calendar month for whatever was chosen in `startDate`. I have tried setting the v-model to the same as startDate (but it still returns to today when opening), and also tried to use the `setCalendarTo` method `endDatePicker.value?.setCalendarTo(start.getFullYear(), start.getMonth());` but not sure I am using it correctly (https://quasar.dev/vue-components/date/). Not sure what I am doing wrong can anyone point me to an example of `setCalendarTo`?

**Comment by metalsadman** [comment, marked as answer]
model should work, and setCalendarTo should as well. https://codepen.io/metalsadman/pen/YzdXPJg?editors=101

**Reply by ssuess** [reply]
Thanks! Not sure which is better, but I ended up using `:default-year-month="endDefaultYearMonth"` on my endDate element with this:
```
const endDefaultYearMonth = computed(() => {
  if (startDate.value) {
    const date = new Date(startDate.value);
    const year = date.getFullYear();
    const month = date.getMonth() + 1; // Months are 0-indexed
    return `${year}/${month.toString().padStart(2, '0')}`; // Formats month as two digits
  }
  return undefined;
});
```
